admin-ui/plugins/fido/components/Fido.tsx

```diff
@@ -35,25 +35,24 @@ const Fido = () => {
   )
 
   const handleStaticConfigurationSubmit = useCallback(
-    (data) => {
+    (data: FormData) => {
       const apiPayload = createFidoConfigPayload({
         fidoConfiguration,
         data,
         type: fidoConstants.STATIC,
       })
-
       dispatch(putFidoConfiguration(apiPayload))
     },
     [dispatch, fidoConfiguration],
   )
 
-  const tabNames = [
+  const tabNames: TabName[] = [
     { name: t('menus.static_configuration'), path: '/fido/fidomanagement/static-configuration' },
     { name: t('menus.dynamic_configuration'), path: '/fido/fidomanagement/dynamic-configuration' },
   ]
 
   const tabToShow = useCallback(
-    (tabName) => {
+    (tabName: string) => {
       switch (tabName) {
```

Comment:
Tighten type of tabName parameter.

Constrain to TabName['name'] for stronger exhaustiveness.

```diff
-    (tabName: string) => {
+    (tabName: TabName['name']) => {
```